content/6_Advanced/Multiplicative.mdx

```diff
@@ -76,40 +73,38 @@ Focus problem 1 requires finding the count of prime numbers not exceeding a give
 ```cpp
 #include <bits/stdc++.h>
 using namespace std;
-constexpr int N(5e8),M(3.2e7);
-int p[M],pc;
+constexpr int N(5e8), M(3.2e7);
+int p[M], pc;
```

Comment:
1. can you just use `=`
2. declarations on separate lines
3. 3.2e7 is such a specific number